tutorials/transfers/index.md

```diff
@@ -0,0 +1,271 @@
+---
+title: Polkadot.Js Guides about Transfers
+description: Tutorials on how to transfer and manage funds on the Polkadot network, covering metadata updates, sending funds, teleporting tokens, calculating fees, and more.
+---
+
+# Polkadot.Js Guides about Transfers
+
+## Metadata Updates
+
+Before you start using the [Polkadot.Js Browser Extension](https://polkadot.js.org/extension/){target=\_blank} to interact with the Polkadot network, it is crucial to ensure that the metadata is updated. The user interface may not function properly if the metadata is not up-to-date.
+
+To update the metadata of the Polkadot.Js Browser Extension, follow these steps:
+
+1. Access the Polkadot.Js App interface and click on the **Settings** tab:
+
+    ![Settings Tab](/images/tutorials/transfers/metadata-update/metadata-update-1.webp)
+
+2. Click on the **Metadata** tab:
+
+    ![Metadata Tab](/images/tutorials/transfers/metadata-update/metadata-update-2.webp)
+
+3. If there are any updates available, click on the **Update metadata** button:
+
+    ![Update Metadata Button](/images/tutorials/transfers/metadata-update/metadata-update-3.webp)
+
+4. A pop-up window will appear to confirm the update. Click on the **Yes, I do this metadata update** button to allow the extension to update the metadata:
+
+    ![Update Metadata Confirmation](/images/tutorials/transfers/metadata-update/metadata-update-4.webp)
+
+Now that the metadata is updated, you can proceed with transferring and managing funds on the Polkadot network.
+
+## Transfers
+
+### Sending Funds
+
+Whether your account is in the Polkadot Browser Extension or created/added directly on the Polkadot-JS UI, you must use the Polkadot-JS UI interface to send funds or issue any extrinsic. The Polkadot extension is an account manager, not a wallet, requiring a UI to interact with.
```

Comment:
This is just a way of doing it, you could also transfer funds directly using wallets' built-in features, but this is only true for the polkadot.js browser extension. I would remove `must` and mention that other methods exist